contracts/pool/src/error.rs

```diff
@@ -29,4 +29,5 @@ pub enum ContractError {
     SlippageInvalid = 20,
 
     SwapMinReceivedBiggerThanReturn = 21,
+    NotEnoughSharesToBeMinted = 22,
```

Comment:
What does "Not enough shares" refers to?

```suggestion
    NotEnoughLiquidityProvided = 22,
```